package.json

```diff
@@ -73,6 +111,18 @@
           ],
           "default": "ask",
           "description": "Controls how NPL instruction files for AI assistants are handled"
+        },
+        "noumena.cloud.portalUrl": {
+          "type": "string",
+          "default": "https://portal.pre.noumena.cloud",
+          "description": "Noumena Cloud portal URL",
+          "scope": "resource"
+        },
+        "noumena.cloud.keycloakUrl": {
+          "type": "string",
+          "default": "https://keycloak.pre.noumena.cloud",
+          "description": "Noumena Cloud Keycloak URL",
+          "scope": "resource"
```

Comment:
we don't really want the users to modify these, but for testing this will be invaluable.